tools/docker/iroha-all-in-one/entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -e
+
+# sync the local config file with the environment variables for pg opts
+export pg_opt_value="host=${IROHA_POSTGRES_HOST} port=${IROHA_POSTGRES_PORT} user=${IROHA_POSTGRES_USER} password=${IROHA_POSTGRES_PASSWORD}"
+
+jq --arg pg_opt "${pg_opt_value}" \
+    '.pg_opt = $pg_opt' \
+    config.docker|sponge config.docker
+
+# if first arg looks like a flag, assume we want to run irohad server
+if [ "${1:0:1}" = '-' ]; then
+  set -- irohad "$@"
+fi
+
+if [ "$1" = 'irohad' ]; then
+  echo key=$KEY
+  echo $PWD
+  if [ -n "$IROHA_POSTGRES_HOST" ]; then
+    echo "NOTE: IROHA_POSTGRES_HOST should match 'host' option in config file"
+    PG_PORT=${IROHA_POSTGRES_PORT:-5432}
+    /wait-for-it.sh -h $IROHA_POSTGRES_HOST -p $PG_PORT -t 30 -- true
```

Comment:
is the path of the script correct: `/wait-for-it.sh ` - should it be here `/`?

Reply:
Yes! The Iroha team copied wait-for-it.sh to absolute path `/`.
[Reference](https://github.com/hyperledger/iroha/blob/main/docker/release/Dockerfile#L14)

Reply:
OK, thanks for explaination. So the path is OK if iroha is doing the same thing:)